misk/src/test/kotlin/misk/web/interceptors/ConcurrencyLimitsInterceptorTest.kt

```diff
@@ -216,7 +216,7 @@ class ConcurrencyLimitsInterceptorTest {
 
   private fun callSuccessCount(id: String): Double {
     return prometheusRegistry.getSampleValue(
-      "concurrency_limits_outcomes",
+      "concurrency_limits_outcomes_total",
```

Comment:
If the Prometheus bump requires downstream metrics name changes, should service owners expect to need to update downstream CI alert formulas (ie in SignalFX, Datadog...) given changed metric names?

Reply:
Yes.  According to prometheus: https://github.com/prometheus/client_java/releases/tag/parent-0.10.0, this was done to done to match OpenMetrics.